dockstore-integration-testing/src/test/java/io/dockstore/client/cli/CheckerWorkflowIT.java

```diff
@@ -168,6 +168,14 @@ public void testCWLToolAddCheckerRefreshPublishUnpublish() throws ApiException {
         final long count9 = testingPostgres
             .runSelectStatement("select count(*) from tool where ispublished = true", long.class);
         assertEquals("the tool should not be published, there are " + count9, 0, count9);
+
+        // Should not be able to directly publish the checker
+        try {
+            workflowApi.publish(refreshedEntry.getCheckerId(), publishRequest);
+            assertTrue("Should not reach this statement.", false);
```

Comment:
You can also Assert.fail();

Reply:
Intellij has something like a "Simplify all assert tests" operation